solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -666,20 +654,18 @@ contract StandardPoolConverter is ConverterVersion, IConverter, ContractRegistry
         assert(amount < targetBalance);
 
         // ensure that the input amount was already deposited
-        uint256 actualSourceBalance;
-        if (_sourceToken == NATIVE_TOKEN_ADDRESS) {
-            actualSourceBalance = address(this).balance;
+        uint256 actualSourceBalance = _sourceToken.balanceOf(address(this));
```

Comment:
the fact that we have 2 conditional expressions here instead of one slightly increases the gas price but should be very very low, right?

Reply:
Yeah 🙏